static_src/components/complex_list.jsx

```diff
@@ -50,16 +42,16 @@ export default class ComplexList extends React.Component {
         title = this.props.title;
       }
       header = (
-        <header className={ this.styler('complex_list-header') }>
-          <h4 className={ this.styler('complex_list-title') }>
+        <header className="complex_list-header">
+          <h4 className="complex_list-title">
             { title }
           </h4>
         </header>
       );
     }
 
     return (
-      <div className={ [this.props.className, this.styler('complex_list')].join(' ') }>
+      <div className={ [this.props.className, 'complex_list'].join(' ') }>
```

Comment:
Use `classNames` to do the joining? Also because `this.props.className` might be falsey so this could render `undefined complex_list` right?